### PR TITLE
Package torch.0.2

### PR DESCRIPTION
**Pull request by LaurentMazare**
### `torch.0.2`
PyTorch bindings for OCaml
The ocaml-torch project provides some OCaml bindings for the PyTorch library.
This brings to OCaml NumPy-like tensor computations with GPU acceleration and
tape-based automatic differentiation.

---
* Homepage: https://github.com/LaurentMazare/ocaml-torch
* Source repo: git+https://github.com/LaurentMazare/ocaml-torch.git
* Bug tracker: https://github.com/LaurentMazare/ocaml-torch/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
You didn't define the archive for the release so opam doesn't know where to get it

File: packages/torch/torch.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+run-test: [
+  ["dune" "runtest" "-p" name "-j" jobs]
+]
+
+depends: [
+  "base" {>= "0.11.0"}
+  "cmdliner"
+  "ctypes" {>= "0.5"}
+  "ctypes-foreign"
+  "dune" {>= "1.3.0" build}
```
Comment (kit-ty-kate) [inline packages/torch/torch.0.2/opam]:
```suggestion
  "dune" {>= "1.3.0" & build}
```

**Comment by LaurentMazare** [comment]
Indeed, I didn't understand why CI was failing - thanks for investigating. I'll add the archive information when I get a chance.
One weird thing is that I used 'opam publish' to create the PR so I would have hoped for it to properly populate this section.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**